src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/Index/required_fields.html.twig

```diff
@@ -34,6 +34,7 @@
         <div class="alert-text">
           <p>{{ 'Select the fields you would like to be required for this section.'|trans({}, 'Admin.Orderscustomers.Help') }}</p>
           <p>{{ 'Please make sure you are complying with the opt-in legislation applicable in your country.'|trans({}, 'Admin.Orderscustomers.Help') }}</p>
+          <p>{{ "Warning: setting a checkbox to 'required' will require users, from back-office and front-office, to check it."|trans({}, 'Admin.Orderscustomers.Help') }}</p>
```

Comment:
@LouiseBonnard I try to explain this https://github.com/PrestaShop/PrestaShop/issues/17817#issuecomment-591514972

But it's hard with so few words 😅 . The idea is to explain the confusion between a "required to be checked" checkbox and a "mandatory/necessary" field (like birthdate).

Reply:
```suggestion
          <p>{{ 'Note that setting an option as required enjoins users to complete it, whether it is on the back office or the front office.'|trans({}, 'Admin.Orderscustomers.Help') }}</p>
```

Reply:
It is clearer with this formulation? You tell me. :-)